test/geocel/g4/GeantGeo.test.cc

```diff
@@ -446,222 +317,18 @@ TEST_F(SolidsTest, output)
             to_string(out));
     }
 }
-
 //---------------------------------------------------------------------------//
 
-TEST_F(SolidsTest, trace)
+TEST_F(SolidsTest, accessors)
 {
-    if (CELERITAS_UNITS != CELERITAS_UNITS_SI)
-    {
-        /*
-         * Note: with SI, this trace skips over the sphere! (During
```

Comment:
Was this resolved or removed from testing?

Reply:
Resolved; the difference was because this was along the edge of a surface.